src/main.js

```diff
@@ -164,10 +166,16 @@ module.exports = {
               }
             }
             let range
+            const msgLine = line - 1
+            const msgCol = column ? column - 1 : column
             try {
-              range = Helpers.rangeFromLineNumber(
-                textEditor, line - 1, column ? column - 1 : column
-              )
+              if (endColumn && endLine) {
```

Comment:
Are we sure that a valid `endColumn` won't be `0`?  If we get an endColumn of zero, and an `endLine` of, say, `5`, it seems like we should still use them.  But I think this check will return false, right?  Perhaps a safer thing to do is check `typeof endColumn !== 'undefined'`?